content/blogposts/2024/cody-vscode-1-10-0-release.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Cody for VS Code v1.10.0 release"
+authors:
+  - name: Kalan Chan
+    url: https://handbook.sourcegraph.com/team/#kalan-chan
+publishDate: 2024-03-20T10:00-01:00
+description: "Cody for VS Code v1.10.0 is now available, this release includes support for Claude 3 Haiku, several improvements to doc string generation, and debugging."
+tags: [blog]
+slug: "cody-vscode-1-10-0-release"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/cody-vscode-1.10.0-og-image.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/cody-vscode-1.10.0-og-image.png
+--- 
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v1.10.0 is now available, this release includes support for Claude 3 Haiku, several improvements to doc string generation, and debugging. 
+
+
+## Support links for Cody Pro and Enterprise
+
+We have made it easier for Cody Pro and Enterprise users to get assistance when needed, support links are now more prominently displayed so users can easily access help. Cody Pro and Enterprise users can now get more timely responses by filling out a support form. 
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/support1.png"
+  alt="Support link in the settings panel"
+/>
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/support2.png"
+  alt="Support link in the Cody settings panel"
+/>
+
+## Improved Document Command
+
+Generating documentation from code has been improved to automatically detect documentable code blocks and provide hints to quickly generate doc strings. When your cursor is on a documentable symbol in JavaScript, TypeScript, Go or Python, a "Opt/Alt+D to Document" ghost text will appear. Pressing Opt/Alt+D will immediately generate documentation for that code block.
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/document-command.png"
+  alt="Document Command"
+/>
+
+## Claude 3 Haiku is now available for Cody Pro 
+
+Cody Pro users now have access to Anthropic’s latest AI model - Claude 3 Haiku. This new model offers significantly faster speeds while maintaining quality and coherence. Claude 3 Haiku is available in the LLM selection dropdown, read more about Claude 3 Haiku [here](https://www.anthropic.com/news/claude-3-haiku). 
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/claude3-haiku.png"
+  alt="Claude3 Haiku in the LLM Selection dropdown"
+/>
+
+## Better Debugging for Cody
+
+We've added new commands and shortcuts to make debugging Cody easier, these improvements enable debugging and accessing logs. 
+
+- Debug commands now grouped in command palette under "Cody Debug"
+- New sidebar menu item to open the output channel
+- New settings menu button to enable debug mode and output channel
```

Comment:
```suggestion
- Debug commands are now grouped in command palette under "Cody Debug"
- Added an "Open Output Channel" sidebar menu item, that opens the VS Code output panel with "Cody by Sourcegraph" selected
- Added a debug button in the top right of the Cody settings panel, which sets `{ "cody.debug.verbose": true }` and opens the VS Code output panel
```

^ @abeatrix is that correct

^ (also why do we have two ways of doing the same thing, and that they're slightly different?)

Reply:
The sidebar is not available to users who can't / haven't logged in, is that what you're referring to (sidebar vs menu)?